.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
+package-lock.json
```

Comment:
I'd rather not have that in here because changes should be made with pnpm so they also update the lockfile. Else package.json and the shrinkwrap go out of sync